charts/logging-operator/values.yaml

```diff
@@ -239,15 +239,38 @@ logging:
   #           type: json
   #         default_topic: topic
 
-  # -- EventTailer config
-  eventTailer: {}
-    # name: sample
-    # pvc:
-    #   accessModes:
-    #     - ReadWriteOnce
-    #   volumeMode: Filesystem
-  #   storage: 1Gi
-  #   storageClassName: standard
+  # EventTailer config
+  eventTailer:
+    create: false
```

Comment:
nit: I would prefer enabled to be consistent

Reply:
done